api/android/api/build.gradle

```diff
@@ -45,6 +45,9 @@ android {
         }
     }
     buildTypes {
+        debug {
```

Comment:
Looks. good. 

@jaeyun-jung  BTW, does the existing "**TAOS/pr-audit-nnstreamer-apptest**" module check the change of the *.java code of this PR?

Reply:
@leemgs  CI just checks build failure.

Reply:
@jaeyun-jung  Synched.

Reply:
@jaeyun-jung , Can we reproduce the build and run test with the existing guide (e.g., README.md)?
* https://github.com/nnsuite/nnstreamer/blob/master/api/android/README.md

Reply:
Yes, we can. The test result and library file will be copied into ```android_lib``` directory.